webpack/components/OpenscapRemediationWizard/steps/ReviewRemediation.js

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable camelcase */
+import React, { useContext, useState } from 'react';
+import { some } from 'lodash';
+import {
+  CodeBlock,
+  CodeBlockAction,
+  CodeBlockCode,
+  ClipboardCopyButton,
+  Button,
+  Grid,
+  GridItem,
+  Alert,
+  Checkbox,
+} from '@patternfly/react-core';
+import ExternalLinkSquareAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-square-alt-icon';
+
+import { sprintf, translate as __ } from 'foremanReact/common/I18n';
+import { foremanUrl } from 'foremanReact/common/helpers';
+import { getHostsPageUrl } from 'foremanReact/Root/Context/ForemanContext';
+
+import OpenscapRemediationWizardContext from '../OpenscapRemediationWizardContext';
+import WizardHeader from '../WizardHeader';
+import { HOSTS_PATH, FAIL_RULE_SEARCH } from '../constants';
+import { findFixBySnippet, reviewHostCount } from '../helpers';
+
+import './ReviewRemediation.scss';
+
+const ReviewRemediation = () => {
+  const {
+    fixes,
+    snippet,
+    method,
+    hostName,
+    source,
+    isRebootSelected,
+    setIsRebootSelected,
+    hostIdsParam,
+  } = useContext(OpenscapRemediationWizardContext);
+  const [copied, setCopied] = useState(false);
+  const selectedFix = findFixBySnippet(fixes, snippet);
+  const snippetText = selectedFix?.full_text;
+  // can be one of null, "true", "false"
+  // if null, it may indicate that reboot might be needed
+  const checkForReboot = () => !some(fixes, { reboot: 'false' });
+  const isRebootRequired = () => some(fixes, { reboot: 'true' });
+
+  const copyToClipboard = (e, text) => {
+    navigator.clipboard.writeText(text.toString());
```

Comment:
(Also relevant to all other copyToClipboards we have), we can assume were always in a in secure context (HTTPS), right?  https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard

Reply:
Well, we should. At least with self-signed certificates.

Reply:
Great!